bin/ncm-cli.js

```diff
@@ -40,9 +41,16 @@ async function main () {
     }
   })
 
-  let [ command = 'help', ...subargs ] = argv._
+  let [ command = 'report', ...subargs ] = argv._
   if (!Object.keys(commands).includes(command)) {
-    command = 'help'
+    try {
+      // alias `ncm DIR` to `ncm report DIR`
+      fs.statSync(command)
+      subargs.unshift(command)
+      command = 'report'
+    } catch (_) {
+      command = 'help'
```

Comment:
The `report` command's internals should be the one handling this.

Also:
```js
const { stat } = require('fs')
const { promisify } = require('util')
const statP = promisify(stat)

// ...

await stat(...)
```

Reply:
> The report command's internals should be the one handling this.

Which part? As far as I understand this logic, then `report` would be required to run the `help` command, if that stat fails. And it's cleaner if commands don't run each other I think.

> Also:

What's the advantage of using async over sync fs calls in a CLI?